stubs/docutils/docutils/nodes.pyi

```diff
@@ -10,8 +11,27 @@ class Node:
     def __getattr__(self, __name: str) -> Any: ...  # incomplete
 
 class Element(Node):
+    children: list[Node]
     def __init__(self, rawsource: str = ..., *children: Node, **attributes): ...
     def __getattr__(self, __name: str) -> Any: ...  # incomplete
+    def __len__(self) -> int: ...
+    def __contains__(self, key: str | Node) -> bool: ...
+    @overload
+    def __getitem__(self, key: str) -> Any: ...
```

Comment:
Why does this return `Any`? What's an attribute in docutils?

Reply:
The attributes are strings mapped to arbitrary objects.